projects/plugins/jetpack/extensions/editor.js

```diff
@@ -18,6 +19,8 @@ import './store/membership-products';
 import extensionList from './index.json';
 import './index.scss';
 
+initializeNumberFormatters();
```

Comment:
I need someone with better knowledge of Jetpack to help me understand if what I'm doing here is correct/enough. The number-formatters package will replace both `@automattic/format-currency` and the [numberFormat](https://github.com/Automattic/jetpack/blob/trunk/projects/js-packages/components/components/number-format/README.md) component/utility.

Hi @anomiex / @jeherve :-)

Reply:
Looks like it won't work as written because that `initializeNumberFormatters` function isn't in the package (and shouldn't be). You'll need to `import { initializeNumberFormatters } from './shared/number-formatters-init.ts'` I think.

Other than that, I don't know enough about this part of the code to say if it's correct/enough either. Sorry. 🙁

Reply:
Ok. Taking a different stab at this, doing a chain of fallbacks:

1. locale set on module
2. locale found in user settings `@wordpress/date` (assuming this is correct)
3. locale found on `window.navigator.language`
4. fallback locale defined in the module "en" (or prospectively "en-US")

Reply:
Good question @jeherve 

Not sure I have an answer. I am mostly interested in what would make most sense for Jetpack. I am going by what the current `numberFormat` in components and `format-currency` package use, which is the locale from user settings or the browser's. 

The other two cases I'm looking at (Calypso and Landpack) will be setting this through `setLocale`, so they won't be resolving to the default for the most part ("for the most part" because packages developed in wp-calypso, if used externally, may end up using the fallbacks as well).

Let me know what you think.

Reply:
> I am going by what the current `numberFormat` in components and `format-currency` package use, which is the locale from user settings or the browser's.

I think that's fine. 👍 It may be worth making that clear in the component's readme so folks who will try to use `@automattic/number-formatters` will know what to expect.

Reply:
Sounds good. Added a generous note about this near the top under `Usage`.

Reply:
Think I will also move this bit to a separate PR and add a test. It will probably be easier to find/reference down the line, if need be. I'll link to our discussion here.

Thanks for discussion :-)

Reply:
Moved and referenced discussion: https://github.com/Automattic/jetpack/pull/42872